ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -632,6 +659,22 @@ private class PostgresLedgerDao(
       .toSet
       .map(Ref.Party.assertFromString)
 
+  private val SQL_GET_LEDGER_ENTRIES = SQL(
+    "select * from ledger_entries where ledger_offset>={startInclusive} and ledger_offset<{endExclusive}")
+
+  //TODO we should use paging instead, check if Alpakka can do that?
```

Comment:
I will address this in a follow up PR

Reply:
what do you mean with paging -- chunk the results rather than getting them all at a time?

Reply:
Precisely. Here it's perfectly possible and it's a much better approach. See: https://github.com/digital-asset/daml/pull/641